lib/collection/src/collection_manager/optimizers/segment_optimizer.rs

```diff
@@ -143,6 +156,17 @@ pub trait SegmentOptimizer {
             .copied()
             .unwrap_or(0);
 
+        // Estimate required disk space
+        let required_space = maximal_vector_store_size_bytes;
```

Comment:
I would argue, that required_space should also include the paylaod size.

I propose to take sum of optimizing segments disk size as a reference instead.

Reply:
Sure, but I am confused about calculating payload's total size.